### PR TITLE
added text to clarify goals; revised other text for clarity and grammar

### PR DESCRIPTION
**Pull request by each**
this started out as an attempt to add text clarifing that the proposed
EDNS buffer sizes are not meant as a permanent change to the EDNS
specification, but an agreed-upon default configuration, intended for
use in the absence of better information about MTU, which will prevent
fragmentation on the majority of present-day networks.

in the process, I ended up making a lot of other revisions for clarity
and grammar, and changed the references to 1220 bytes to 1232, since
that seems to be the growing consensus in #125.

(I should probably have broken this into multiple commits but didn't
think of it in time, sorry.)

### CODE REVIEW

File: 2020/index.md [hunk 1]
```diff
@@ -66,100 +66,135 @@ conferences such as [DNS-OARC 30](https://www.dns-oarc.net/oarc30) panel
...
 ----------------------
 
-This web site and some aspects of DNS flag day 2020 are work in progress.
+This web site and some aspects of DNS Flag Day 2020 are works in progress.
 - The _exact date_ for the 2020 DNS Flag Day is not yet determined.
 - **Please note** that the _exact recommended EDNS buffer sizes_ have not
-  been agreed upon, the current ballpark around 1200 (1220, 1232, ...) is
-  to limit the risk of fragmentation in IPv6.
+  yet been agreed upon. The current proposal is to use a value around
+  1200 (1220, 1232, ...) to limit the risk of fragmentation on IPv6.
```
Comment (jelu) [inline 2020/index.md]:
```suggestion
  1232 to limit the risk of fragmentation on IPv6.
```

Comment (each) [reply 2020/index.md]: Thanks!

**Comment by jelu** [comment]
I will merge this mid week unless someone objects, suggest changes or merges it.

ping @oerdnj @pspacek

**Comment by jelu** [comment]
@ncartron @huguei @kidmin @songlinjian Please update translations based on this merge, thanks!

**Pull request merged**